src/meek.jl

```diff
@@ -0,0 +1,104 @@
+# http://proceedings.mlr.press/v89/katz19a/katz19a-supp.pdf
+# https://arxiv.org/pdf/1302.4972.pdf
+function meek_rules!(g)
+    while true 
```

Comment:
@RobertGregg you say this needs a single pass? I think it needs multiple passes?

Reply:
Yes it does need multiple passes 🤦

Reply:
I have it fixed